cogboard-webapp/src/components/widgets/types/LogViewerWidget/Toolbar/QuarantineModal/index.js

```diff
@@ -43,14 +43,15 @@ const QuarantineModal = ({ wid, quarantine }) => {
   };
 
   const toggleChecked = rule => {
-    const endTimestamp = rule.endTimestamp;
-    if (endTimestamp) {
-      const inFuture = moment.utc(endTimestamp * 1000).local() > moment();
-      if (!inFuture) {
-        return { ...rule, checked: true, endTimestamp: null };
-      }
+    const endTimestamp =
+      Number.isInteger(rule.endTimestamp) &&
+      moment.utc(rule.endTimestamp * 1000).local();
```

Comment:
czym jest to `*1000`? nie chcielibyśmy takich magic numberów bez opisu

Reply:
Zamiana sekund na milisekundy, dodano komentarze.